packages/core-modules/contracts/modules/ElectionModule.sol

```diff
@@ -3,24 +3,24 @@ pragma solidity ^0.8.0;
 
 import "@synthetixio/core-contracts/contracts/errors/InitError.sol";
 import "@synthetixio/core-contracts/contracts/ownership/OwnableMixin.sol";
+import "@synthetixio/core-contracts/contracts/initializable/InitializableMixin.sol";
 import "../interfaces/IElectionModule.sol";
 import "../submodules/election/ElectionSchedule.sol";
 import "../submodules/election/ElectionCredentials.sol";
 import "../submodules/election/ElectionVotes.sol";
 import "../submodules/election/ElectionTally.sol";
 
-/// @title Module for electing a council, represented by a set of NFT holders
 contract ElectionModule is
     IElectionModule,
     ElectionSchedule,
     ElectionCredentials,
     ElectionVotes,
     ElectionTally,
-    OwnableMixin
+    OwnableMixin,
+    InitializableMixin
```

Comment:
Bringing InitializableMixin here, so that ElectionBase doesn't impose initializer on ElectionInspectorModule.